modules/apps/27-interchain-accounts/events/emit_events.go

```diff
@@ -0,0 +1,19 @@
+package events
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types"
+)
```

Comment:
Could we rename this file to `events.go` to be consistent with other events in other pkgs?

mega nit: separate imports here also.

```suggestion
import (
	sdk "github.com/cosmos/cosmos-sdk/types"
	
	"github.com/cosmos/ibc-go/v3/modules/apps/27-interchain-accounts/types"
)
```